doc/source/whatsnew/v1.3.3.rst

```diff
@@ -21,6 +21,15 @@ Fixed regressions
 
 .. ---------------------------------------------------------------------------
 
+.. _whatsnew_133.performance:
+
+Performance improvements
+~~~~~~~~~~~~~~~~~~~~~~~~
+- Performance improvement for :meth:`DataFrame.__setitem__` when the key or value is not a :class:`DataFrame`, or key is not list-like (:issue:`43274`)
+-
+-
```

Comment:
We need a blank line, docs aren't building like that

Reply:
ah! I see. I was confused why the docs were consistently failing.